### ISSUE TITLE
[Content styleguide] Content doesn’t align with style guidance in VA.gov content style guide. (05.08.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#68263]
#### VFS team name
Authenticated Experience

#### VFS product name
My VA

#### VFS feature name
Dot indicator

#### Point of Contact/Reviewers
Brian Smolensky - @BSmolensky - Content

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Content doesn’t align with style guidance in VA.gov content style guide.

### Issue Details
>My VA landing page, >Health Care, >Next Appointment Box: In this box, the user finds the Date and Time of thier next apppointment. The time needs to be fixed to align with the style guide. According the the VA design style guide, content writers should put periods between a.m. and p.m. The periods are missing here. 

### VA.gov Experience Standard
[Category Number 05, Issue Number 08](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

### Platform Recommendation
Please fix the time. 

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by ajakabcin** [comment]
Auth Exp My VA team will address this in #68290 and #68293

**Comment by ajakabcin** [comment]
Resolved!
![Screenshot 2023-11-03 at 12 21 23 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/45603961/febd64fd-f82c-48af-9a05-0cc2ebd718a8)
